tests/test_forecasting.py

```diff
@@ -75,7 +75,7 @@ def test_forecasting_GEPS(numeric_config, get_local_testdata):
     # Prepare a RAVEN model run using historical data, GR4JCN in this case.
     # This is a dummy run to get initial states. In a real forecast situation,
     # this run would end on the day before the forecast, but process is the same.
-    wup.start_date = dt.datetime(2000, 1, 1)
```

Comment:
Fine as long as we don't need datetime.date or another object from the module. I've been bitten in the past, hence now suggest using dt for the module and not the class.

Reply:
I think I added this while fixing some tests. This one is worth reverting. Thanks